### PR TITLE
[NG] bare draggable and droppable with demo setup

### PR DESCRIPTION
**Pull request by Shijir**
Bare directives for Draggable and Droppable as well as demo pages for `draggable`, `draggable-ghost` and `draggable-handler` included in this PR. 

This PR will be a foundation for incoming implementations for `ClrDraggable`.

Signed-off-by: Shijir Tsogoo <stsogoo@vmware.com>

### CODE REVIEW

File: src/app/drag-and-drop/drag-and-drop-draggable.demo.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-drag-and-drop-draggable",
+    styleUrls: ["./drag-and-drop.demo.scss"],
+    templateUrl: "./drag-and-drop-draggable.demo.html"
+})
+export class DragAndDropDraggableDemo {}
```
Comment (adityarb88) [inline src/app/drag-and-drop/drag-and-drop-draggable.demo.ts]: Why not just `DraggableDemo`?

File: src/app/drag-and-drop/drag-and-drop-ghost.demo.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-drag-and-drop-ghost",
+    styleUrls: ["./drag-and-drop.demo.scss"],
+    templateUrl: "./drag-and-drop-ghost.demo.html"
+})
+export class DragAndDropGhostDemo {}
```
Comment (adityarb88) [inline src/app/drag-and-drop/drag-and-drop-ghost.demo.ts]: DragGhostDemo?

File: src/app/drag-and-drop/drag-and-drop-handle.demo.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-drag-and-drop-handle",
+    styleUrls: ["./drag-and-drop.demo.scss"],
+    templateUrl: "./drag-and-drop-handle.demo.html"
+})
+export class DragAndDropHandleDemo {}
```
Comment (adityarb88) [inline src/app/drag-and-drop/drag-and-drop-handle.demo.ts]: DragHandleDemo?

File: src/app/drag-and-drop/basic-draggable.demo.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="clr-example">
+    <button class="btn btn-primary">draggable</button>
```
[the remark this answers is not in the document]
Comment (Shijir) [reply src/app/drag-and-drop/basic-draggable.demo.html]: Sure, I will add an example with a non-button element.

**Pull request merged**